packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactBaseTextShadowNode.java

```diff
@@ -221,6 +259,93 @@ private static void buildSpannedFromShadowNode(
     }
   }
 
+  private static void buildSpannedFromShadowNodeUnified(
+      ReactBaseTextShadowNode textShadowNode,
+      SpannableStringBuilder sb,
+      List<SetSpanOperation> ops,
+      @Nullable TextAttributes parentTextAttributes,
+      boolean supportsInlineViews,
+      @Nullable Map<Integer, ReactShadowNode> inlineViews,
+      int start) {
+
+    TextAttributes textAttributes;
+    if (parentTextAttributes != null) {
+      textAttributes = parentTextAttributes.applyChild(textShadowNode.mTextAttributes);
+    } else {
+      textAttributes = textShadowNode.mTextAttributes;
+    }
+
+    final var textAttributeProvider = new HierarchicTextAttributeProvider(textShadowNode, parentTextAttributes, textAttributes);
+
+    for (int i = 0, length = textShadowNode.getChildCount(); i < length; i++) {
+      ReactShadowNode child = textShadowNode.getChildAt(i);
+
+      if (child instanceof ReactRawTextShadowNode) {
+        sTextLayoutUtils.addText(sb, ((ReactRawTextShadowNode) child).getText(), textAttributeProvider);
+      } else if (child instanceof ReactBaseTextShadowNode) {
+        buildSpannedFromShadowNodeUnified(
+            (ReactBaseTextShadowNode) child,
+            sb,
+            ops,
+            textAttributes,
+            supportsInlineViews,
+            inlineViews,
+            sb.length());
+      } else if (child instanceof ReactTextInlineImageShadowNode inlineImageChild) {
```

Comment:
```suggestion
      } else if (child instanceof ReactTextInlineImageShadowNode) {
```

Formatter, but oddly not compiler, caught invalid token here

Reply:
Got it. It looks like the version of `google-java-format` we use for formatting is old enough to not be able to handle the new syntax. Could we use something more compatible?